.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,17 +5,22 @@
 
 ### Fixed Issues
 <!---
-Please replace GH_LINK with the link to the GitHub issue this Pull Request is fixing.
+1. Please replace GH_LINK with a URL link to the GitHub issue this Pull Request is fixing.
```

Comment:
Should we add a note to this that merging the PR will automatically close the issue? 
I've seen some questions about that.

Reply:
It will close it only for internal employees, right? I think in that case it should be fine